src/quo2/components/channel_avatar.cljs

```diff
@@ -0,0 +1,48 @@
+(ns quo2.components.channel-avatar
+  (:require [quo2.foundations.colors :as colors]
+            [quo.react-native :as rn]
+            [status-im.ui.components.icons.icons :as icons]))
+
+(defn channel-avatar [{:keys [big? dark? lock-status icon-color icon x y]}]
```

Comment:
better to confirm with design team, but we don't have icons for channel avatars. These are emojis and background color, and we construct icon ourselves inside the component.

Reply:
ref: https://github.com/status-im/status-mobile/issues/12375

Reply:
Can I use it just as a placeholder ?

Reply:
placeholder, wdym?

Reply:
I see now, You mean these are actual textual emojis and they should have the background color as a prop
GOTCHA!

Reply:
Done, Kindly check